File: CustomDialog/app/src/main/java/com/example/customdialog/MainActivity.java
package com.example.customdialog;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.AppCompatButton;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    AppCompatButton btnClose;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: FacebookAds/app/src/main/java/com/example/facebookads/MainActivity.kt
package com.example.facebookads

import android.os.Bundle
import android.widget.LinearLayout
import androidx.appcompat.app.AppCompatActivity
import com.facebook.ads.AdSize
import com.facebook.ads.AdView
import com.facebook.ads.AudienceNetworkAds

class MainActivity : AppCompatActivity() {

    lateinit var adView: AdView

    override fun onCreate(savedInstanceState: Bundle?) {
        AudienceNetworkAds.initialize(this);
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        initView()
    }

    private fun initView() {

        adView = AdView(this, "IMG_16_9_APP_INSTALL#YOUR_PLACEMENT_ID", AdSize.BANNER_HEIGHT_50)

        val adContainer = findViewById<LinearLayout>(R.id.banner_container) as LinearLayout

        adContainer.addView(adView)

        adView.loadAd()

    }
}
